lib/msal-browser/src/naa/AccountRequests.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export type AccountByHomeIdRequest = {
```

Comment:
Why objects with a single property instead of just using the property directly?

Reply:
Agree. Could bloat the code for no reason.